definitions/modules/safety.yaml

```diff
@@ -6,9 +6,16 @@ shared:
     event_name: TOGGLE_ALTERNATE_STATIC
 
   - # ASOBO_SAFETY_Switch_ELT_Template
-    type: var
+    type: ToggleSwitch
     var_name: A:ELT ACTIVATED
-    var_units: Number
+    var_units: Bool
+    var_type: bool
+    off_event_name: ELT_OFF
+    event_name: ELT_ON
+
+  -  
+    type: var
+    var_name: L:XMLVAR_ELT_STATE
```

Comment:
Haven't tested myself yet but I believe ELT state handles ELT ACTIVATED on it's own. Most switches are 3 states so toggleswitch would override state and mess up sync.

Reply:
From testing, this def is required to sync the ELT.